src/__tests__/Button.test.tsx

```diff
@@ -89,4 +89,51 @@ describe('Button', () => {
     const button = container.getByRole('button')
     expect(button).toMatchSnapshot()
   })
+  it('overrides the styles declared with data attributes with sx prop', () => {
```

Comment:
@joshblack I added tests to target specifically the css attributes that are overridden with the data attributes. Let me know what you think about the tests and if you have any recommendation 🙌🏻